app/ledger/n3NeonLedger.js

```diff
@@ -3,6 +3,14 @@ import * as neonJs from '@cityofzion/neon-js-next'
 import * as n3ledger from '@cityofzion/neon-ledger-next'
 import type { Transaction } from '@cityofzion/neon-js-next'
 import LedgerNode from '@ledgerhq/hw-transport-node-hid'
+import { cloneDeep } from 'lodash-es'
+
+const VALID_STATUS = 0x9000
+const MSG_TOO_BIG = 0x6d08
+const APP_CLOSED = 0x6e00
+const APP_CLOSED_V2 = 0x6e01
+const TX_DENIED = 0x6985
+const TX_PARSE_ERR = 0x6d07
```

Comment:
MSG_TOO_BIG, APP_CLOSED, TX_PARSE_ERR and not valid status words for the NEO3 app, see https://github.com/CityOfZion/neon-js/blob/820e20d3160e46d019c3f65e84d3f9270ffa1588/packages/neon-ledger/src/ErrorCode.ts#L1